lib/gameteam.rb

```diff
@@ -35,8 +35,29 @@ def self.create_from_csv(game_teams_path)
         shots: row["shots"],
         tackles: row["tackles"]
       }
-    game_teams << GameTeam.new(gameteam_data)
+    @@game_teams << GameTeam.new(gameteam_data)
     end
-    game_teams
+    @@game_teams
+  end
+
+  def self.game_teams
+    @@game_teams
+  end
+
+  def self.avg_scores_per_team_home
```

Comment:
There's not test for this in the gameteam_spec file.  I see that there's a test for the integration test in the stat_tracker, but we'll also need one in the gameteam_spec file as well.